storage/transfermanager/integration_test.go

```diff
@@ -295,6 +296,75 @@ func TestIntegration_DownloadDirectoryAsync(t *testing.T) {
 				t.Errorf("unexpected file %q in dir", entry.Name())
 			}
 		}
+
+		// Now attempt to download the entire directory.
+		// The existing files should be skipped.
+		callbacks := make(chan bool)
+
+		d, err = NewDownloader(c, WithWorkers(2), SkipIfExists())
+		if err != nil {
+			t.Fatalf("NewDownloader: %v", err)
+		}
+
+		if err := d.DownloadDirectory(ctx, &DownloadDirectoryInput{
+			Bucket:         tb.bucket,
+			LocalDirectory: localDir,
+			OnObjectDownload: func(got *DownloadOutput) {
+				callbacks <- true
+
+				if got.Err != nil {
+					t.Errorf("result.Err: %v", got.Err)
+				}
+
+				if strings.EqualFold(got.Object, "dir/objA") {
```

Comment:
This only checks `dir/objA`, do you want to check the other included objects?

Reply:
Added.